subprojects/sol2.wrap

```diff
@@ -0,0 +1,11 @@
+[wrap-file]
+directory = sol2-3.3.0
+source_url = https://github.com/ThePhD/sol2/archive/refs/tags/v3.3.0.tar.gz
+source_filename = sol2-3.3.0.tar.gz
+source_hash = b82c5de030e18cb2bcbcefcd5f45afd526920c517a96413f0b59b4332d752a1e
+
+[provide]
+dependency_names = sol2
+
+[patch_file]
+filename = use_wrapdb_lua.patch
```

Comment:
WrapDB does not allow patching upstream sources.  We already have a `lua` wrap that should be picked up automatically.